src/makeCalculator.js

```diff
@@ -4,7 +4,54 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(callback, number) {
+      callback.call(this, number);
+
+      return this;
+    },
+
+    add(number) {
+      return (this.result += number);
+    },
+
+    subtract(number) {
+      this.result -= number;
+    },
+
+    multiply(number) {
+      this.result *= number;
+    },
```

Comment:
The `multiply` method should return `this` to maintain chainability, similar to the `operate` and `reset` methods.